client/js/kernel.js

```diff
@@ -44,18 +55,14 @@ var Services = require('jupyter-js-services');
                     clearTimeout(debounced);
                     debounced = setTimeout(function() {
                         var isBusy = status === 'busy';
-                        $('.busy-indicator')
-                            .toggleClass('show', isBusy)
-                            // Prevent progress animation when hidden by removing 'active' class.
-                            .find('.progress-bar')
-                                .toggleClass('active', isBusy);
+                        showBusyIndicator(isBusy);
                     }, 500);
                 });
                 return kernel;
             })
             .catch(function(e) {
-                console.error('failed to create kernel', e);
-                throw e;
+                showBusyIndicator(false);
+                throw new Error('Failed to create kernel');
```

Comment:
I would change this to:

`throw new Error('Failed to create kernel: ' + e.message)`

This will provide more information.